tests/Testcontainers.Platform.Windows.Tests/WindowsContainerTest.cs

```diff
@@ -11,7 +11,7 @@ private WindowsContainerTest(IContainer container)
 
     public async ValueTask InitializeAsync()
     {
-        await _container.StartAsync()
+        await _container.StartAsync().WaitAsync(TimeSpan.FromSeconds(30))
```

Comment:
```suggestion
        await _container.StartAsync()
```

Is this necessary? Why?